src/test/java/org/sonatype/nexus/ci/config/Nxrm2ConfigurationTest.groovy

```diff
@@ -30,183 +30,14 @@ class Nxrm2ConfigurationTest
   @Rule
   public JenkinsRule jenkins = new JenkinsRule()
 
-  def 'it validates that display name is unique'() {
```

Comment:
Trivial: For IQ we have the implementation tests extend the `IqPolicyEvaluatorDescriptorTest` to ensure that the subclass correctly implemented the superclass (or did not alter the implementation). Imagine a new implementation for `doCheckDisplayName` is added to `Nxrm3Configuration`, right now nothing would break. OTOH it would probably be caught in code review.

Reply:
Also removes the need for `SimpleNxrmConfiguration`